Source/Generator/README.md

```diff
@@ -3,53 +3,40 @@
 This repo contains the LabVIEW Measurement template code
 
 ## Dependency List
+
 1. LabVIEW 2020 64 bit
-2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases)
-This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for,
-2.1 ni_lib_labview_grpc_library-x.x.x.x.vip
-2.2 ni_lib_labview_grpc_servicer-x.x.x.x.vip
+2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases) This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for
```

Comment:
Don't we include grpc-labview support in our releases? If so, I would mention to install those versions instead since we know they will work.

Reply:
Referenced the ones included in our release.